sw/device/mask_rom/boot.md

```diff
@@ -95,52 +110,87 @@ void boot(void) {
     // undo the changes, for instance to write-enable bits).
 
     // System State Measurements (2.c.iv)
-    measurements = perform_system_state_measurements();
-    if (!boot_allowed(boot_policy, measurements)) {
+    measurements = perform_system_state_measurements(); // System State Module
+    if (!boot_allowed(boot_policy, measurements)) { // System State Module
```

Comment:
Why is this in system state module instead of boot policy?

Reply:
I don't think the "correct" measurements will be stored with the boot policy in flash, they'll probably be hardcoded for a single chip. 

Looking at the boot policy structure, I'm not sure it has anything which will change the result of this function, so maybe this doesn't take boot_policy.